src/main/webapp/app/guided-tour/guided-tour.component.ts

```diff
@@ -84,11 +83,15 @@ export class GuidedTourComponent implements AfterViewInit, OnDestroy {
         }
     }
 
+    private isTourStepType(type: TextLinkTourStep | ImageTourStep | VideoTourStep): boolean {
```

Comment:
Two things:
- The function is not used, so I would propose removing it until we need it.
- This is something that is pretty bad in Typescript. I normally try to use enums for type checking in situations like this.

Reply:
@thilo-behnke You're right, I totally forgot to remove this function